user_modules/hmi_values.lua

```diff
@@ -0,0 +1,425 @@
+        phoneCapability = {
+          dialNumberEnabled = true
+        }
+      }
+    },
+    mandatory = true,
+    pinned = false
+  }
+
+  hmi_table.VR.GetCapabilities = {
+    params = {
+      vrCapabilities = { "TEXT" }
+    },
+    mandatory = true,
+    pinned = false
+  }
+
+  hmi_table.TTS.GetCapabilities = {
+    params = {
+      speechCapabilities = { "TEXT", "PRE_RECORDED" },
+      prerecordedSpeechCapabilities = {
+        "HELP_JINGLE", "INITIAL_JINGLE", "LISTEN_JINGLE", "POSITIVE_JINGLE","NEGATIVE_JINGLE"
+      }
+    },
+    mandatory = true,
+    pinned = false
+  }
+
+  hmi_table.Buttons.GetCapabilities = {
+    params = {
+      capabilities = {
+        module.createButtonCapability("PRESET_0"),
+        module.createButtonCapability("PRESET_1"),
+        module.createButtonCapability("PRESET_2"),
+        module.createButtonCapability("PRESET_3"),
+        module.createButtonCapability("PRESET_4"),
+        module.createButtonCapability("PRESET_5"),
+        module.createButtonCapability("PRESET_6"),
+        module.createButtonCapability("PRESET_7"),
+        module.createButtonCapability("PRESET_8"),
+        module.createButtonCapability("PRESET_9"),
+        module.createButtonCapability("OK", true, false, true),
+        module.createButtonCapability("SEEKLEFT"),
+        module.createButtonCapability("SEEKRIGHT"),
+        module.createButtonCapability("TUNEUP"),
+        module.createButtonCapability("TUNEDOWN")
+      },
+      presetBankCapabilities = { onScreenPresetsAvailable = true },
+    },
+    mandatory = true,
+    pinned = false
+  }
+
+  hmi_table.RC.GetCapabilities = {
+    params = {
+      remoteControlCapability = {
+        climateControlCapabilities = {
+            {
+              moduleName = "Climate",
+              fanSpeedAvailable = true,
+              desiredTemperatureAvailable = true,
+              acEnableAvailable = true,
+              acMaxEnableAvailable = true,
+              circulateAirEnableAvailable = true,
+              autoModeEnableAvailable = true,
+              dualModeEnableAvailable = true,
+              defrostZoneAvailable = true,
+              defrostZone = {
+                "FRONT", "REAR", "ALL", "NONE"
+              },
+              ventilationModeAvailable = true,
+              ventilationMode = {
+                "UPPER", "LOWER", "BOTH", "NONE"
+              }
+            }
+          },
+        radioControlCapabilities = {
+            {
+              moduleName = "Radio",
+              radioEnableAvailable = true,
+              radioBandAvailable = true,
+              radioFrequencyAvailable = true,
+              hdChannelAvailable = true,
+              rdsDataAvailable = true,
+              availableHDsAvailable = true,
+              stateAvailable = true,
+              signalStrengthAvailable = true,
+              signalChangeThresholdAvailable = true
+            }
+          },
+        buttonCapabilities = function()
```

Comment:
@dboltovskyi According HMI_API buttonCapabilities is array of ButtonCapabilities, actual it is function. 
Please fix.